console.go

```diff
@@ -312,6 +312,11 @@ func needsQuote(s string) bool {
 
 // colorize returns the string s wrapped in ANSI code c, unless disabled is true.
 func colorize(s interface{}, c int, disabled bool) string {
+	e := os.Getenv("NO_COLOR")
+	if e == "true" {
```

Comment:
[GNU libtextstyle](https://www.gnu.org/software/gettext/libtextstyle/manual/html_node/The-NO_005fCOLOR-variable.html) doc says:
> The environment variable NO_COLOR can be used to suppress styling in the textual output. When this environment variable is set (to any value), libtextstyle-enabled programs will not emit colors and other text styling.

Reply:
Ahh you are right, fixed the code now.